src/modules/uniplayer/Uniplayer.jsx

```diff
@@ -170,26 +170,28 @@ class Uniplayer extends React.Component {
 		return (
 			<div className={'uniplayer' + player.active} onMouseMove={this.playerActive}>
 				<SearchContainer />
-				<div className="song-lines">
-					<div className="line-box">
-						<input
-							className="player-bar"
-							ref="playerBar"
-							type="range"
-							min={0}
-							max={1}
-							step="any"
-							value={this.state.played}
-							onMouseEnter={this.onMouseEnter}
-							onMouseMove={this.onMouseMove}
-							onMouseLeave={this.onMouseLeave}
-							onMouseDown={this.onSeekMouseDown}
-							onChange={this.onSeekChange}
-							onMouseUp={this.onSeekMouseUp}
-						/>
-					</div>
-				</div>
 				<div className="player-holder">
+					<div className="song-lines">
```

Comment:
This should really be its own component. It is an isolated piece of UI that does a fixed thing. We should create a component called `SongLine` that takes renders one and has all the styling, and then create a container `SongLines` that can take an arbitrary number of `SongLine`s as children.

`Uniplayer` is too big, we need to start breaking it up.

Reply:
Sounds good, I'll take a stab at it right now

Reply:
On question, because the song line acts like the Playerbar (updating playback). It needs to directly affect the state of the Uniplayer component. Would you envision doing that through props or a global state like Redux?

Reply:
For now we can do it by passing bound callbacks into the `SongLines` (pretty much the way you're doing it now) but soon we'll want all of the players state to be in Redux. That way any part of the app can change what the players doing without needing to talk to the player directly.

Reply:
You'll find that when we really start using Redux, very very little (if any state) will be stored in `this.state`

Reply:
Ok nice, I'm going to refactor this PR and simplify some of it, removing the need for two separate playback bars. 

Let me take a stab at that and then will follow up

Also, would love to work through setting up the Redux store with you and learn how you intend on architecting it!